### ISSUE TITLE
Back-to-top element has no element.style when scrolling up

### ISSUE THREAD

**Issue by lampensau** [squidfunk/mkdocs-material#3283]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

When scrolling upwards the javascript removes the `top: 131px` element style of the button. 

This breaks the positioning when activating `navigation.tabs.sticky` as the button is partially hidden behind the navigation tabs.

### Expected behaviour

The element style for `top: ` on the back-to-top button should not be removed when scrolling upwards.

### Actual behaviour

![grafik](https://user-images.githubusercontent.com/12431219/144244121-2f242bbb-c83a-47ed-9511-24482c122506.png)

Potential cause: https://github.com/squidfunk/mkdocs-material/blob/79c56f58c4afcd69fe72a4fc10b85801917d2bfb/src/assets/javascripts/components/top/index.ts#L150

### Steps to reproduce

Build documentation with the following features:

```yaml
  features:
    - navigation.tabs.sticky
    - navigation.top
```

### Package versions

- Python: 3.9.2
- MkDocs: 1.2.3
- Material: 8.0.2+insiders.4.1.0

### Configuration

```yaml
site_name: My Docs

theme:
  name: material
  features:
    - navigation.tabs.sticky
    - navigation.top
```

### System information

- Browser: Chrome, Firefox, Safari

**Comment by squidfunk** [comment]
Fixed in `f8a19dc06` – I split the pipeline.

**Comment by squidfunk** [comment]
Released as part of 8.0.3
